financialaid/models.py

```diff
@@ -171,8 +182,8 @@ def save_and_log(self, acting_user, *args, **kwargs):
         FinancialAidAudit.objects.create(
             acting_user=acting_user,
             financial_aid=self,
-            data_before=serializers.serialize("json", [financialaid_before, ]),
-            data_after=serializers.serialize("json", [self, ])
+            data_before=financialaid_before.to_dict(),
```

Comment:
A DoesNotExist error would only come from a bug.  There are three ways one could happen:

* Someone calls `save_and_log` on a FinancialAid object that has not yet been saved to the database
* Someone calls `save_and_log` on a FinancialAid object that has been deleted
* Someone modifies the `id` field and then calls `save_and_log`

None of these are correct uses of the FinancialAid object